{"cvrfdoc":{"document_title":"Red Hat Security Advisory: OpenShift Container Platform 4.4.9 python-psutil security update","document_distribution":"Copyright Â© 2021 Red Hat, Inc. All rights reserved.","document_references":{"reference":[{"description":"https://access.redhat.com/errata/RHSA-2020:2583","type":"Self","url":"https://access.redhat.com/errata/RHSA-2020:2583"},{"description":"https://access.redhat.com/security/updates/classification/#moderate","type":"External","url":"https://access.redhat.com/security/updates/classification/#moderate"}]},"aggregate_severity":"Moderate","document_tracking":{"initial_release_date":"2020-06-22T23:50:00Z","identification":{"id":"RHSA-2020:2583"},"revision_history":{"revision":{"date":"2020-06-22T23:50:00Z","number":"1","description":"Current version"}},"generator":{"date":"2021-08-23T04:45:00Z","engine":"Red Hat SDEngine 3.0.0"},"current_release_date":"2020-06-22T23:50:00Z","version":"1","status":"Final"},"product_tree":{"relationship":[{"relates_to_product_reference":"7Server-RH7-RHOSE-4.4","product_reference":"python-psutil-0:5.6.6-1.el7ar","full_product_name":{"product_id":"7Server-RH7-RHOSE-4.4:python-psutil-0:5.6.6-1.el7ar","cpe":"cpe:/a:redhat:openshift:4.4::el7","product_name":"python-psutil-0:5.6.6-1.el7ar as a component of Red Hat OpenShift Container Platform 4.4"},"relation_type":"Default Component Of"}],"branch":[{"name":"Red Hat OpenShift Enterprise","type":"Product Family","branch":[{"name":"Red Hat OpenShift Container Platform 4.4","full_product_name":{"product_id":"7Server-RH7-RHOSE-4.4","cpe":"cpe:/a:redhat:openshift:4.4::el7","product_name":"Red Hat OpenShift Container Platform 4.4"},"type":"Product Name"}]},{"name":"python-psutil-0:5.6.6-1.el7ar","full_product_name":{"product_id":"python-psutil-0:5.6.6-1.el7ar","cpe":"cpe:/a:redhat:openshift:4.4::el7","product_name":"python-psutil-5.6.6-1.el7ar.src.rpm"},"type":"Product Version"}]},"document_publisher":{"issuing_authority":"Red Hat Product Security","contact_details":"secalert@redhat.com","type":"Vendor"},"vulnerability":[{"notes":{"note":"python-psutil: double free because of refcount mishandling"},"cve":"CVE-2019-18874","references":{"reference":[{"description":"CVE-2019-18874","url":"https://access.redhat.com/security/cve/CVE-2019-18874"},{"description":"bz#1772014: CVE-2019-18874 python-psutil: double free because of refcount mishandling","url":"https://bugzilla.redhat.com/show_bug.cgi?id=1772014"}]},"release_date":"2019-11-07T00:00:00Z","involvements":{"involvement":{"party":"Vendor","status":"Completed"}},"product_statuses":{"status":{"product_id":["7Server-RH7-RHOSE-4.4:python-psutil-0:5.6.6-1.el7ar"],"type":"Fixed"}},"remediations":{"remediation":{"description":"For OpenShift Container Platform 4.4 see the following documentation, which\nwill be updated shortly for release 4.4.9, for important instructions on\nhow to upgrade your cluster and fully apply this asynchronous errata\nupdate:\nhttps://docs.openshift.com/container-platform/4.4/release_notes/ocp-4-4-release-notes.html\nDetails on how to access this content are available at\nhttps://docs.openshift.com/container-platform/4.4/updating/updating-cluster-cli.html.","type":"Vendor Fix","url":"https://access.redhat.com/errata/RHSA-2020:2583"}},"threats":{"threat":{"description":"Moderate","type":"Impact"}},"acknowledgments":{"acknowledgment":{"description":"This issue was discovered by Riccardo Schirone (Red Hat)."}},"discovery_date":"2019-11-13T00:00:00Z","ordinal":"1"}],"document_notes":{"note":["An update for python-psutil is now available for Red Hat OpenShift Container Platform 4.4.\nRed Hat Product Security has rated this update as having a security impact of Moderate. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.","Red Hat OpenShift Container Platform is Red Hat's cloud computing\nKubernetes application platform solution designed for on-premise or private\ncloud deployments.\nSecurity Fix(es):\n* python-psutil: A double free issue has been discovered in python-psutil that may allow a local attacker to get code execution with the privileges of the user running the python-psutil application. (CVE-2019-18874)\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.","This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original."]},"document_type":"Security Advisory"}}